settings/LoanPolicy/components/DeleteEntity/DeleteEntity.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  ConfirmationModal,
+} from '@folio/stripes/components';
+
+import { IfPermission } from '@folio/stripes/core';
+
+class DeleteEntity extends React.Component {
+  static propTypes = {
+    policyName: PropTypes.string.isRequired,
+    initialValues: PropTypes.object,
+    confirmDelete: PropTypes.bool.isRequired,
+    changeDeleteState: PropTypes.func.isRequired,
+    onRemove: PropTypes.func.isRequired,
+  };
+
+  onConfirm = () => {
+    const {
+      initialValues,
+      onRemove,
+    } = this.props;
+
+    onRemove(initialValues);
+  };
+
+  onCancel = () => {
+    this.props.changeDeleteState(false);
+  };
+
+  openConfirmationModal = () => {
+    this.props.changeDeleteState(true);
+  }
+
+  render() {
+    const {
+      confirmDelete,
+      policyName,
+    } = this.props;
+
+    const confirmationMessage = <FormattedMessage
+      id="ui-circulation.settings.loanPolicy.deleteMessage"
+      values={{ name: <strong>{policyName}</strong>, removed: <strong>removed</strong> }}
```

Comment:
"removed" won't be translated if we write it this way. Use `<SafeHTMLMessage>` instead, like in EntryForm.js

Reply:
I have fixed that. Thank you.